airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java

```diff
@@ -30,7 +30,8 @@
 /**
  * Record Consumer used for STANDARD INSERTS
  */
-public class BigQueryRecordConsumer extends FailureTrackingAirbyteMessageConsumer implements AirbyteMessageConsumer {
+@SuppressWarnings("try")
+class BigQueryRecordConsumer extends FailureTrackingAirbyteMessageConsumer implements AirbyteMessageConsumer {
```

Comment:
The problem here is that the throw of AirbyteMessageConsumer throws an Exception, which could be an InterruptedException, and that triggers a warning